### PR TITLE
Added missing import

### PR DESCRIPTION
**Pull request by GrahamCampbell**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

#### What's in this PR?

Adds a missing import to fix fatal errors.

#### Why?

image

#### Checklist

- [ ] Updated CHANGELOG.md to describe BC breaks / deprecations | new feature | bugfix
- [ ] Documentation pull request created (if not simply a bugfix)

#### To Do

- [ ] If the PR is not complete but you want to discuss the approach, list what remains to be done here

### CODE REVIEW

File: src/Plugin/HistoryPlugin.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 namespace Http\Client\Common\Plugin;
 
+use Exception;
```
Comment (sagikazarmark) [inline src/Plugin/HistoryPlugin.php]: Conflicts with the other exception import. Also, our convention is not to import root namespaces classes, but use them directly.

**Comment by sagikazarmark** [comment]
This might not be that easy though. The Journal only accepts Httplug exceptions, so we need to check the exception type in the rejection handler.

@Nyholm you created this part AFAIK. What was the convention? The Journal only collects HTTP related exceptions, right? (So we can show them in the debug toolbar?) Any other exception is out of scope IMO.

**Comment by sagikazarmark** [comment]
Thanks @GrahamCampbell. So Styleci uses HTTPlug now? 😄 Exciting. Our code is used to test our code?

**Comment by GrahamCampbell** [comment]
Hi, yeh, the link is https://github.com/KnpLabs/php-github-api/issues/409. I was testing the new library on StyleCI (now in production). If we can't get this resolved soonish, I will need to rollback the deployment.

**Comment by GrahamCampbell** [comment]
How does this look now, I've taken off the typehint so we can deal with any eventuality, and added an instanceof check.

**Comment by GrahamCampbell** [comment]
Hmm, maybe we don't actually need to change anything here. Looks like the other library is actually broken here:

image

Throwing another exception there should not be allowed, right?

**Comment by GrahamCampbell** [comment]
Closing this. There is no bug in your library. The other library is just misusing it, right?

**Comment by GrahamCampbell** [comment]
How does https://github.com/KnpLabs/php-github-api/pull/410 look to you please?

---

Thanks for the cool library btw. :)

**Pull request closed without merging**